src/index.js

```
@@ -48,6 +51,16 @@ const createPost = body => {
   return post
 }
 
+const convertId = _id => {
```

This function is useless and bad. It swallows an exception.